PyIRI/main_library.py

```diff
@@ -1281,7 +1308,7 @@ def gammaE(year, mth, utime, alon, alat, aIG):
     return gamma_E, solzen_out, solzen_eff_out, slon, slat
 
 
-def Probability_F1(year, mth, utime, alon, alat, mag_dip_lat, aIG, foE):
+def Probability_F1(year, mth, utime, alon, alat):
```

Comment:
Changing the inputs and outputs need to be adjusted so that the old style can be used until a subsequent release.  We don't want to break people's existing code.